packages/core/admin/admin/src/render.ts

```diff
@@ -7,6 +7,13 @@ import { StrapiApp, StrapiAppConstructorArgs } from './StrapiApp';
 interface RenderAdminArgs {
   customisations: StrapiAppConstructorArgs['adminConfig'];
   plugins: StrapiAppConstructorArgs['appPlugins'];
+  features?: {
+    [key: string]: unknown;
+    future?: {
+      unstable_contentReleases: boolean;
+      [key: string]: unknown;
+    };
+  };
```

Comment:
I don't like too much the idea of redeclare this again here but I didn't find a way to share this type with createBuildContext file

Reply:
We will have this config strongly typed in a few weeks for v5 with the env var loading, finally merging in the configuration file types that have been stalled for a couple months and creating a source of truth for all config files.

Is this just for the front-end build to be aware of it? Either way, I think it's better to be explicit. We don't want anyone internally making use of undocumented flags, and we don't want user code piggybacking on this.

Reply:
Lets make it explicit and when that config work is merged we can update this ⭐